### ISSUE TITLE
all loss is nan

### ISSUE THREAD

**Issue by vanson1104** [lzx1413/PytorchSSD#109]
Training FSSD_vgg on VOC0712
/home/edabk/anaconda3/envs/vanson/lib/python3.8/site-packages/torch/nn/functional.py:780: UserWarning: Note that order of the arguments: ceil_mode and return_indices will changeto match the args list in nn.MaxPool2d in a future release.
  warnings.warn("Note that order of the arguments: ceil_mode and return_indices will change"
Epoch:1 || epochiter: 0/15|| Totel iter 0 || L: nan C: nan||Batch time: 5.3048 sec. ||LR: 0.00001000
Epoch:1 || epochiter: 10/15|| Totel iter 10 || L: nan C: nan||Batch time: 0.4946 sec. ||LR: 0.00001000
Epoch:2 || epochiter: 5/15|| Totel iter 20 || L: nan C: nan||Batch time: 0.4915 sec. ||LR: 0.00001000
Epoch:3 || epochiter: 0/15|| Totel iter 30 || L: nan C: nan||Batch time: 1.7345 sec. ||LR: 0.00001000
Epoch:3 || epochiter: 10/15|| Totel iter 40 || L: nan C: nan||Batch time: 0.5051 sec. ||LR: 0.00001000
Epoch:4 || epochiter: 5/15|| Totel iter 50 || L: nan C: nan||Batch time: 0.5323 sec. ||LR: 0.00001000
Epoch:5 || epochiter: 0/15|| Totel iter 60 || L: nan C: nan||Batch time: 1.8704 sec. ||LR: 0.00001000
Epoch:5 || epochiter: 10/15|| Totel iter 70 || L: nan C: nan||Batch time: 0.4923 sec. ||LR: 0.00001000
Epoch:6 || epochiter: 5/15|| Totel iter 80 || L: nan C: nan||Batch time: 0.5012 sec. ||LR: 0.00001000
Epoch:7 || epochiter: 0/15|| Totel iter 90 || L: nan C: nan||Batch time: 1.5594 sec. ||LR: 0.00001000
Epoch:7 || epochiter: 10/15|| Totel iter 100 || L: nan C: nan||Batch time: 0.4778 sec. ||LR: 0.00001000
Epoch:8 || epochiter: 5/15|| Totel iter 110 || L: nan C: nan||Batch time: 0.5084 sec. ||LR: 0.00001000
Epoch:9 || epochiter: 0/15|| Totel iter 120 || L: nan C: nan||Batch time: 1.7057 sec. ||LR: 0.00001000
Epoch:9 || epochiter: 10/15|| Totel iter 130 || L: nan C: nan||Batch time: 0.5020 sec. ||LR: 0.00001000
Epoch:10 || epochiter: 5/15|| Totel iter 140 || L: nan C: nan||Batch time: 0.5000 sec. ||LR: 0.00001000
